src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -90,3 +94,12 @@ def acr_import(cmd,
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
+
+
+def _validate_target_tags(target_tags):
+    if not target_tags:
+        return
+
+    for t_tag in target_tags:
```

Comment:
Simple validation to deal with the user-input issue @SteveLasker found. cc: @sajayantony @djyou 

Does this look good or is this overkill @sajayantony 

![target tags warning](https://user-images.githubusercontent.com/15149427/63982784-c1ee0a80-ca78-11e9-9e8a-3b12e7b51040.png)

Reply:
I would recommend having another PR for the import issue and let this one focus on cross tenant login.

Reply:
Done!